homeassistant/components/apprise/notify.py

```diff
@@ -66,6 +70,20 @@ def send_message(self, message="", **kwargs):
         However, if any tags are specified, then they will be applied
         to the notification causing filtering (if set up that way).
         """
+
         targets = kwargs.get(ATTR_TARGET)
         title = kwargs.get(ATTR_TITLE, ATTR_TITLE_DEFAULT)
-        self.apprise.notify(body=message, title=title, tag=targets)
+
+        data = kwargs.get(ATTR_DATA)
+        attachments = data.get(ATTR_ATTACHMENTS) if data else None
+
+        # Default Attachment
+        attach = None
+        if attachments:
+            # Support file attachments (such as security camera images,
+            # documents, icons, etc). The attachments can be a list and/or
+            # a string.  The AppriseAttachment() object accepts both
+            attach = apprise.AppriseAttachment()
+            attach.add(attachments)
```

Comment:
You need to verify that the attachments are in whitelisted paths using `hass.config.is_allowed_path(…)`

Reply:
Done!